### ISSUE TITLE
Enhancement: [Slider] Add option to show slider value on hover

### ISSUE THREAD

**Issue by macandcheese** [Esri/calcite-components#1125]
### Description
It would be nice to have an option to show the value of a slider in a tooltip for cases where the persistent value doesn't need to be there

@ddred11 can you attach the screenshot of your current use case? Thanks!

### Acceptance Criteria 
Add an opt-in prop that shows the value of a slider on hover. Determine which prop "wins" between persisted value display and "show in tooltip on hover" (or, is both in that case ok?)

**Comment by ddred11** [comment]
@macandcheese Here is a screen shot of a current slider experience we have in BA web. When we adopt the calcite component we would like to have this option...

<img width="338" alt="Screen Shot 2020-10-14 at 3 58 59 PM" src="https://user-images.githubusercontent.com/5332496/96481241-1f4e9900-11f0-11eb-8d3c-33dbead84af0.png">

**Comment by macandcheese** [comment]
cc @bstifle for reference (note - it would just use our current calcite tooltip on hover). I think this is a good addition!

**Comment by bstifle** [comment]
Dig it!!!

**Comment by ddred11** [comment]
@macandcheese  for our enterprise version, admins can change the dominant color of the app to match their branding. Does this component (and I guess most of the others) allow for this type of customization?

**Comment by macandcheese** [comment]
Hey @ddred11 - yep, see this documentation page and playground: https://esri.github.io/calcite-components/?path=/story/overview-custom-theme--interactive-example

Currently you would replace the "blue" CSS variables with the desired app's branding. We may add another "brand" set of CSS variables so we can maintain blue for use cases like "info" status in the future.

**Comment by macandcheese** [comment]
Still valid.

**Comment by macandcheese** [comment]
Valid

**Comment by ddred11** [comment]
@macandcheese Would be nice to add this, as we merge this component into BA. Thanks

**Comment by macandcheese** [comment]
Agreed, maybe a good one for @caripizza ?

**Comment by caripizza** [comment]
@macandcheese sure! 👍

**Comment by macandcheese** [comment]
Keep open

**Comment by macandcheese** [comment]
Valid

**Comment by macandcheese** [comment]
Note - ensure this shows on focus as well.

**Comment by jcfranco** [comment]
If the implementation of this ends up using popper or `calcite-popover`, it may be blocked by https://github.com/Esri/calcite-components/issues/3815.

**Comment by geospatialem** [comment]
Consider as a native tooltip with an option, like https://github.com/Esri/calcite-design-system/issues/1125#issuecomment-712293942 or similar?

**Comment by ashetland** [comment]
Could couple with #2991
